aspnetcore/data/ef-rp/read-related-data.md

```diff
@@ -31,57 +32,87 @@ The following illustrations show the completed pages for this tutorial:
 
-Consider a query that is currently returning a list, such as `ToList` or `Single`. Queries returning a list don't include related data. To include related data, use the `Load` method instead of a method that returns a list.
+ ![Separate queries example](read-related-data/_static/separate-queries.png)
 
-  ![Separate queries example](read-related-data/_static/separate-queries.png)
-  
+ Note: EF Core automatically fixes up navigation properties to any other entities that were previously loaded into the context instance. Even if the data for a navigation property is *not* explicitly included, the property may still be populated if some or all of the related entities were previously loaded.
 
-* Explicit loading. When the entity is first read, related data isn't retrieved. Code is written that retrieves the related data if it's needed. As in the case of eager loading with separate queries, explicit loading results in multiple queries sent to the DB. The difference is that with explicit loading, the code specifies the navigation properties to be loaded. In Entity Framework Core 1.1 you can use the `Load` method to do explicit loading. For example:
+* [Explicit loading](https://docs.microsoft.com/ef/core/querying/related-data#explicit-loading). When the entity is first read, related data isn't retrieved. Code must be written to retrieve the related data when it's needed. Explicit loading with separate queries results in multiple queries sent to the DB. With explicit loading, the code specifies the navigation properties to be loaded. Use the `Load` method to do explicit loading. For example:
 
-  ![Explicit loading example](read-related-data/_static/explicit-loading.png)
+ ![Explicit loading example](read-related-data/_static/explicit-loading.png)
 
-* Lazy loading. When the entity is first read, related data isn't retrieved. However, the first time you attempt to access a navigation property, the data required for that navigation property is automatically retrieved. A query is sent to the DB each time you try to get data from a navigation property for the first time. Entity Framework Core 1.0 does not support lazy loading.
+* [Lazy loading](https://docs.microsoft.com/ef/core/querying/related-data#lazy-loading). [EF Core does not currently support lazy loading](https://github.com/aspnet/EntityFrameworkCore/issues/3797). When the entity is first read, related data isn't retrieved. The first time a navigation property is accessed, the data required for that navigation property is automatically retrieved. A query is sent to the DB each time a navigation property is accessed for the first time.
+
+* The `Select` operator loads only the related data needed, and is generally faster than using `Include`. [Loading related data with Select](#select) explains this in more detail.
```

Comment:
"is generally faster than using Include" @divega @anpete Is this true?

Reply:
Not in general. Up until recently projecting navs would often mean n+1 queries, and it still does for sub-queries rooted on collection navs (@maumar is working on it). I guess the case where it could be is for non-collection navs that are shaped down to project only a subset of the related data.

Reply:
Perhaps the statement came from an EF6-based tutorial?